ui/components/access-table.js

```diff
@@ -1,69 +1,369 @@
-import { sortByPrivilege, sortBySubject } from '../lib/grants'
-import dayjs from 'dayjs'
+import { ChevronDownIcon, XIcon, CheckIcon } from '@heroicons/react/solid'
```

Comment:
If this component is only used in one page, we should move it into that file (vs exporting it in a separate file'